crates/sui-indexer-alt-framework/src/pipeline/concurrent/collector.rs

```diff
@@ -213,8 +213,8 @@ mod tests {
         const FIELD_COUNT: usize = 32;
     }
 
+    use crate::db::Db;
```

Comment:
I had not spotted these imports hanging out here before -- they belong with the rest of the imports, up top.